src/DB/Query/ShippingRateQuery.php

```diff
@@ -67,4 +67,27 @@ function ( $row ) {
 			$this->results
 		);
 	}
+
+	/**
+	 * Get all shipping rates.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @return array
+	 */
+	public function get_all_shipping_rates() {
+		$rates = $this->get_results();
+		$items = [];
+		foreach ( $rates as $rate ) {
+			$data = [
+				'country_code'            => $rate['country'],
+				'currency'                => $rate['rate'],
+				'free_shipping_threshold' => $rate['options']['free_shipping_threshold'] ?: $rate['options']['free_shipping_threshold'],
```

Comment:
Was this supposed to fallback to null or 0?

Reply:
I made it fall back to `null`, I can make it more explicit in the code.